Create home_3_3.go printSorted

opi-dopi opened the pull request.

opi-dopi committed:
Create home_3_3.go

imatsiash-ib commented on home_3_3.go:
```diff
@@ -0,0 +1,33 @@
+		newMap[v] = x[v]
+	}
+
+	return newMap
```
According to the task description this function should only print the map values. You should not to return map.

opi-dopi committed:
Update home_3_3.go

imatsiash-ib commented on home_3_3.go:
```diff
@@ -0,0 +1,33 @@
+
+	for _, v := range arKeys {
+
+		newMap[v] = x[v]
```
You shouldn't create a new map. Just print values.

burov reviewed (changes requested):
The main purpose of this task to teach you have to traverse map in order by keys way, please implement task correctly

opi-dopi committed:
Update home_3_3.go

I hope I understand right last comments or i need do this task without sort.Ints.
